vscode/src/azure/workspaceActions.ts

```diff
@@ -301,6 +301,21 @@ export async function submitJob(
 
   const jobName = await vscode.window.showInputBox({ prompt: "Job name" });
 
+  // validator for the user-provided number of shots input
+  const validateShotsInput = (input: string) => {
+    const result = parseFloat(input);
```

Comment:
Why not use parseInt if you're going to check for that? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt